prisma/schema/voyage.prisma

```diff
@@ -14,9 +14,10 @@ model Voyage {
   createdAt DateTime @default(now()) @db.Timestamptz()
   updatedAt DateTime @updatedAt
 
-  voyageTeams        VoyageTeam[]
-  sprints            Sprint[]
-  voyageApplications VoyageApplication[]
+  voyageTeams          VoyageTeam[]
+  sprints              Sprint[]
+  voyageApplications   VoyageApplication[]
+  DefaultVoyageProject DefaultVoyageProject[]
```

Comment:
please use camelCase instead of PascalCase for `defaultVoyageProject` just to be consistent with the rest of the code